### PR TITLE
entrypoint: clean up logging and indentation

### PR DESCRIPTION
**Pull request by BenTheElder**

### CODE REVIEW

**Review by neolit123 — COMMENTED** [review]
/lgtm

File: images/base/files/usr/local/bin/entrypoint [hunk 1]
```diff
@@ -18,13 +18,24 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+# logging helpers
+log_info() {
+  echo "INFO: $1" >&2
```
Comment (aojea) [inline images/base/files/usr/local/bin/entrypoint:23]: we were not logging info to stderr, is this change on purpose?

Comment (BenTheElder) [reply images/base/files/usr/local/bin/entrypoint:23]: We mostly were (which is correct for diagnostic messages), but not always
